Source/VisualRecognitionV4/VisualRecognition.swift

```diff
@@ -161,17 +161,16 @@ public class VisualRecognition {
     {
         // construct body
         let multipartFormData = MultipartFormData()
-
-        // HAND EDIT: join collectionIDs into CSV string
-        if let csvCollectionIDsData = collectionIDs.joined(separator: ",").data(using: .utf8) {
-            multipartFormData.append(csvCollectionIDsData, withName: "collection_ids")
+        for item in collectionIDs {
+            if let itemData = item.data(using: .utf8) {
+                multipartFormData.append(itemData, withName: "collection_ids")
+            }
         }
-
-        // HAND EDIT: join features into CSV string
-        if let csvFeaturesData = features.joined(separator: ",").data(using: .utf8) {
-            multipartFormData.append(csvFeaturesData, withName: "features")
+        for item in features {
+            if let itemData = item.data(using: .utf8) {
+                multipartFormData.append(itemData, withName: "features")
+            }
```

Comment:
Are we sure this hand edit is no longer required?

Reply:
I missed a hand edit in `cherry-pick.` This should be resolved now.